src/super_gradients/training/utils/ema_decay_schedules.py

```diff
@@ -0,0 +1,38 @@
+import math
+from abc import abstractmethod
+
+__all__ = ["IDecayFunction", "ConstantDecay", "ThresholdDecay", "ExpDecay", "EMA_DECAY_FUNCTIONS"]
+
+
+class IDecayFunction:
+    @abstractmethod
+    def __call__(self, decay: float, step: int, total_steps: int):
```

Comment:
please add a documentation here, in case someone want to write his own decay function